eclair-core/src/main/scala/fr/acinq/eclair/api/JsonSerializers.scala

```diff
@@ -186,4 +186,15 @@ object JsonSupport extends Json4sSupport {
 
   implicit val shouldWritePretty: ShouldWritePretty = ShouldWritePretty.True
 
+  case class CustomTypeHints(custom: Map[Class[_], String]) extends TypeHints {
```

Comment:
Can this be put in a separate file?

Reply:
I think it belongs here logically, it's where we define our application-wide JsonSupport and all the custom serializers.